common/Resources/ti.internal/extensions/node/internal/errors.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Node's internal/errors module modified for Axway Titanium
+ *
+ * Only a few selected errors are exported manually here. Most of the functionality
+ * is still missing and may be added as we move forward with Node compatibility.
+ *
+ * @see https://github.com/nodejs/node/blob/master/lib/internal/errors.js
+ */
+
+import assert from './assert';
+import { format } from './util/inspect';
+
+const messages = new Map();
+export const codes = {};
+
+// @todo implement this once needed
+class SystemError extends Error {
+
+}
+
+// Utility function for registering the error codes.
+function E(sym, val, def, ...otherClasses) {
+	// Special case for SystemError that formats the error message differently
+	// The SystemErrors only have SystemError as their base classes.
+	messages.set(sym, val);
+
+	if (def === SystemError) {
+		throw new Error('Node compatible SystemError not yet implemented.');
+	} else {
+		def = makeNodeErrorWithCode(def, sym);
+	}
+
+	if (otherClasses.length !== 0) {
+		otherClasses.forEach((clazz) => {
```

Comment:
Yeah, me too, but since this was also taken from the node source, i tried to make only the neccessary changes to make our linting pass (for example, wrap single line if-statements in brackets) and leave everything else as is.